autoload/neomake/signs.vim

```diff
@@ -136,6 +136,15 @@ function! neomake#signs#PlaceVisibleSigns() abort
         if empty(s:sign_queue[type][buf])
             unlet s:sign_queue[type][buf]
         endif
+        if g:neomake_place_signs_immediately
```

Comment:
It also could be skipped in case another one was displayed above already, right?